features/support/helpers.rb

```diff
@@ -44,6 +44,36 @@ def stub_request_with_address(address, body = nil)
   end
 end
 
+
+PoltergeistProc = -> (page, key, value) do
+  page.driver.set_cookie(key, value)
+end
+
+SeleniumProc = -> (page, key, value) do
+  page.driver.browser.manage.add_cookie(name: key, value: value)
+end
+
+RackProc = -> (_page, key, value) do
+  headers = {}
+  Rack::Utils.set_cookie_header!(headers, key, value)
+  cookie_string = headers['Set-Cookie']
+  Capybara.current_session.driver.browser.set_cookie(cookie_string)
+end
+
+WebkitProc = -> (_page, key, value) do
+  headers = {}
+  Rack::Utils.set_cookie_header!(headers, key, value)
+  cookie_string = headers['Set-Cookie']
+  Capybara.current_session.driver.browser.set_cookie(cookie_string)
+end
```

Comment:
Does everything still work if you make `WebkitProc` just an alias of `RackProc`, i.e. `WebkitProc = RackProc`?